charts/osm/values.yaml

```diff
@@ -118,6 +132,9 @@ OpenServiceMesh:
   # -- Deploy Prometheus with OSM installation
   deployPrometheus: false
 
+  # -- Deploy kube-state-metrics with OSM installation
+  deployKubeStateMetrics: false
```

Comment:
Any reason this should be optional?

Reply:
The user may want to bring their own kube-state-metrics rather than have the deployment of the service be managed by OSM.

Reply:
Sounds good. One thing I would change is to introduce this param as a part of a new key called `metrics` in the values.yaml file.

Reply:
I think its a good idea to make the deployment of this optional like promtheus, grafana. I agree with Shashank, a new key called metrics would be a good idea. 

```
metrics : 
  enable : true/false
```